packages/remix-react/components.tsx

```diff
@@ -707,7 +707,7 @@ export function Meta() {
 
     let routeModule = routeModules[routeId];
 
-    if (routeModule.meta) {
+    if (routeModule?.meta) {
```

Comment:
Now that we have "Routes" in our application that do not have a module, we do have to be defensive when accessing module information

Reply:
I don't like how easy this is to miss - TS doesn't yell at us when accessing `routeModule.meta` non-defensively, so I may want to look a bit closer at the types there and see what options we have

Reply:
Update `RouteModules` type to include `| undefined`